iot-e2e-tests/android/things/build.gradle

```diff
@@ -71,7 +71,7 @@ dependencies {
     implementation 'com.android.support.constraint:constraint-layout:1.1.3'
     implementation 'com.android.support:multidex:1.0.3'
 
-    implementation ('com.microsoft.azure.sdk.iot:iot-e2e-common:0.20.2'){
+    implementation ('com.microsoft.azure.sdk.iot:iot-e2e-common:0.21.0'){
         exclude module: 'junit'
```

Comment:
Why was this not incremented during release?

Reply:
will need to update bump version script from internal repo